src/components/pages/career-details/career-form/career-file-upload-type.tsx

```diff
@@ -82,8 +82,13 @@ export const CareerFileUploadType = ({
   watch,
 }: CareerFileUploadTypeProps) => {
   const { t } = useTranslation();
-  const fileName = file.name.split('.');
-  const name = `category_select.${fileName[0]}`;
+  const fileName = file.name.split('.')[0];
+  /**
+   * Create a sanitized file id by replacing anything that's not a letter which can be safely used as an identifier.
```

Comment:
*"letter or digit"
Instead of this comment, why not call the variable `sanitizedFileId`, which would make parts of the comment obsolete?